templates/vue-starter-template/app/app.vue

```diff
@@ -1,12 +1,76 @@
 <script setup lang="ts">
 import type { Schemas } from "#shopware";
+import { getPrefix } from "../i18n/src/helpers/prefix";
 
 const { apiClient } = useShopwareContext();
 const sessionContextData = ref<Schemas["SalesChannelContext"]>();
 const contextResponse = await apiClient.invoke("readContext get /context");
 sessionContextData.value = contextResponse.data;
 
 useSessionContext(sessionContextData.value);
+
+const { locale, availableLocales, defaultLocale, localeProperties, messages } =
+  useI18n();
+const router = useRouter();
+const {
+  getAvailableLanguages,
+  getLanguageCodeFromId,
+  getLanguageIdFromCode,
+  changeLanguage,
+  languages: storeLanguages,
+} = useInternationalization();
+const { languageIdChain, refreshSessionContext } = useSessionContext();
+
+const { data: languages } = await useAsyncData("languages", async () => {
```

Comment:
do we need to block SSR for this? It's additional >100ms for the server response. Proposed solutions

1. load it asynchronously, unless it's necessary for rendering, then
2. load it at the build time and store as static JSON in the project files

Reply:
`Promise.all` used to speed up